CONTRIBUTING.md

```diff
@@ -56,9 +56,30 @@ git clone https://github.com/<UserName>/machine-learning.git
 ```
 
 > **Warning**
->
+> 
 > **There will be slight differences on different systems**
 
+### Configure GIT-SSH links
+Before you push, it's best to use SSH links to make sure the network is connected properly.For more details,please refer to [video](https://www.bilibili.com/video/BV1gF411h7bA?vd_source=b3ed65613de5a4cc19d2fa2ba5651e79).
```

Comment:
Missing space before For. I see similar issues several places. Please double check all your content.

Reply:
Change 'best' to better